underfs/obs/src/main/java/alluxio/underfs/obs/ObsClientExt.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.underfs.obs;
+
+import com.obs.services.ObsClient;
+
+import java.util.Map;
+
+/**
+ * An extension of ObsClient to support config.
+ */
+public class ObsClientExt extends ObsClient {
+
+  /**
+   * Construct obs client.
+   *
+   * @param accessKey ak in the access key secretKey
+   * @param secretKey sk in the access key endPoint
+   * @param endPoint OBS endpoint
+   * @param conf the map of OBS configuration
+   */
+  public ObsClientExt(String accessKey, String secretKey, String endPoint,
+      Map<String, Object> conf) {
+    super(accessKey, secretKey, endPoint);
+    for (Map.Entry<String, Object> entry : conf.entrySet()) {
+      obsProperties.setProperty(entry.getKey(),
+          entry.getValue() == null ? null : entry.getValue().toString());
```

Comment:
Do you think we need to print a DEBUG log here to show what configurations to be set in obsProperties?

Reply:
@Jackson-Wang-7 Done